### DISCUSSION TITLE
how do i work this

### DISCUSSION THREAD

**Discussion by Time2Skip** [3kh0/ext-remover#521] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
![image](https://user-images.githubusercontent.com/119337709/218851360-8151afe1-a834-41a2-821c-ffa3e04b5ff7.png)

**Comment by qDread-dev** [comment, marked as answer]
You put the link to the extension icon in the first box
In the second, put the name of the extension
In the third, the extension id

You can get the extension ID, Icon, and name by going to the chrome web store
If its blocked, add your personal google  account and go to chrome.google.com/webstore/?authuser=1

An example for what to put could be:

https://lh3.googleusercontent.com/m347I3-CgRx0vHEb3Dkws0awrYnRzy0z-CB2nlIlXK8cFyY3Ax1zl_jLZZVTFK84Cq2tDWS2u353-LPwyGHuNu1l9A=w128-h128-e365-rj-sc0x00ffffff
Scientific Calculator
fnplhdldnhaodknidfddmkfdhlhjihpd


To get the extension id copy everything after /webstore/detail/extnamehere 
If you used ?authuser=1 then dont copy that

**Reply by obkduishinijf** [reply]
@ookookokkokoookokoko mark as answer, he explained it better than i could
